pkg/eval/unix/umask.go

```diff
@@ -0,0 +1,94 @@
+// +build !windows,!plan9
+
+package unix
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"strconv"
+	"syscall"
+
+	"github.com/elves/elvish/pkg/eval"
+	"github.com/elves/elvish/pkg/eval/vars"
+)
+
+//elvdoc:var umask
+//
+// The file system creation mask. When used in a string context the output is
```

Comment:
* POSIX calls this "file mode creation mask", let's stick to that.

* Elvish does not have "string context". Just say "its value is a string in octal representation".